CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## Unreleased
+
+- Include hostname in redis cache keys  [#98](https://github.com/microsoft/planetary-computer-apis/pull/98)
+
+## [2022.2.0]
```

Comment:
We tagged 2022.2.0, but hadn't created the GitHub release and changed this changelog - updated here.